x-pack/plugins/session_view/public/components/process_tree_alert/translations.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+
+export const ALERT_TYPE_TOOLTIP = (alertDetailTextTooltipContent: string) =>
+  i18n.translate('xpack.sessionView.alertTypeTooltip', {
```

Comment:
Does this output the expected entries in the compiled language json files? Passing the entire string contents as a parameter like this seems like an anti pattern. Normally those template params are used for dynamic content like counts. e.g   'View {count} alerts'

Each tooltip would also need it's own id, e.g xpack.sessionView.fileAlertTooltip     networkAlertTooltip etc...

Might be easiest to just declare 3 separate translations and have your funciton return the appropriate instance for the type.